From 6978d77d90c37458a7b48770ef622597d4692bcc Mon Sep 17 00:00:00 2001
From: Robert Lancaster <rlancaste@gmail.com>
Date: Tue, 8 Feb 2022 06:38:57 +0000
Subject: [PATCH] Removing WCS Coord disabling line

Removing WCS Coord disable line because I removed it entirely from stellarsolver now.
---
 kstars/ekos/align/align.cpp     | 4 ----
 kstars/tools/horizonmanager.cpp | 2 ++
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/kstars/ekos/align/align.cpp b/kstars/ekos/align/align.cpp
index 859ac6b01..d2b31a240 100644
--- a/kstars/ekos/align/align.cpp
+++ b/kstars/ekos/align/align.cpp
@@ -2011,10 +2011,6 @@ void Align::startSolving()
             m_StellarSolver->setSSLogLevel(SSolver::LOG_OFF);
         }
 
-        //Unless we decide to load the WCS Coord, let's turn it off.
-        //Be sure to set this to true instead if we want WCS from the solve.
-        m_StellarSolver->setLoadWCS(false);
-
         // Start solving process
         m_StellarSolver->start();
     }
diff --git a/kstars/tools/horizonmanager.cpp b/kstars/tools/horizonmanager.cpp
index 232b72f35..b214090f5 100644
--- a/kstars/tools/horizonmanager.cpp
+++ b/kstars/tools/horizonmanager.cpp
@@ -303,6 +303,8 @@ void HorizonManager::slotToggleCeiling()
     if(!regionItem)
         return;
 
+    if(!regionItem)
+        return;
     bool turnCeilingOn = !regionItem->data(Qt::UserRole).toBool();
     if (turnCeilingOn)
     {
-- 
GitLab

